tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Controllers/UsersControllerTests.cs

```diff
@@ -304,12 +307,52 @@ public async Task DisableUser_UserRepository_UpdateAsync_CalledOnce()
             var context = UsersControllerTestContext.Setup();
             using var controller = context.Controller;
 
-            context.User = ApplicationUserBuilder.Create().WithDisabled(false).BuildBuyer();
+            context.User = ApplicationUserBuilder.Create().WithDisabled(false).Build();
 
             await controller.DisableUserAsync(context.User.Id);
 
-            context.UsersRepositoryMock.Verify(x => x.GetUserByIdAsync(context.User.Id), Times.Once);
+            context.UsersRepositoryMock.Verify(x => x.GetByIdAsync(context.User.Id), Times.Once);
             context.UsersRepositoryMock.Verify(x => x.UpdateAsync(context.User), Times.Once);
         }
     }
+
+    internal sealed class UsersControllerTestContext
+    {
+        private UsersControllerTestContext()
+        {
+            CreateBuyerServiceMock = new Mock<ICreateBuyerService>();
+            CreateBuyerServiceMock.Setup(x => x.CreateAsync(It.IsAny<CreateBuyerRequest>()))
+                .ReturnsAsync(() => CreateBuyerResult);
+            
+            UsersRepositoryMock = new Mock<IUsersRepository>();
+
+            Users = new List<ApplicationUser>();
+            UsersRepositoryMock.Setup(x => x.FindByOrganisationIdAsync(It.IsAny<Guid>()))
+                .ReturnsAsync(() => Users);
+
+            UsersRepositoryMock.Setup(x => x.CreateUserAsync(It.IsAny<ApplicationUser>()));
+
+            Controller = new UsersController(CreateBuyerServiceMock.Object, UsersRepositoryMock.Object);
+
+            UsersRepositoryMock.Setup(x => x.GetByIdAsync(It.IsAny<string>())).ReturnsAsync(() => User);
+
+            UsersRepositoryMock.Setup(x => x.UpdateAsync(It.IsAny<ApplicationUser>()));
+        }
+        public Mock<ICreateBuyerService> CreateBuyerServiceMock { get; set; }
```

Comment:
Just missing white space between `}` and the `CreateBuyerServiceMock` property.

Reply:
👍